.github/workflows/regenerate-protobuf.yml

```diff
@@ -0,0 +1,94 @@
+name: Regenerate Protobuf Files
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  regenerate-protobuf:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Assume aws gati role
+        if: github.actor != 'app-token-issuer-engops[bot]'
+        uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4.0.2
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_ARN_GATI }}
+          role-duration-seconds: 900
+          aws-region: ${{ secrets.AWS_REGION }}
+          mask-aws-account-id: true
+
+      - name: Get github token from gati
+        id: get-gh-token
+        if: github.actor != 'app-token-issuer-engops[bot]'
+        uses: smartcontractkit/chainlink-github-actions/github-app-token-issuer@main
+        with:
+          url: ${{ secrets.AWS_LAMBDA_URL_GATI }}
+
+      - name: Checkout repo
+        if: github.actor != 'app-token-issuer-engops[bot]'
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+        # Include the pull request ref in the checkout action to prevent merge commit
+        # https://github.com/actions/checkout?tab=readme-ov-file#checkout-pull-request-head-commit-instead-of-merge-commit
+          ref: ${{ github.event.pull_request.head.sha }}
+          token: ${{ steps.get-gh-token.outputs.access-token }}
+          fetch-depth: 1
+
+      - name: Set up Go
+        if: github.actor != 'app-token-issuer-engops[bot]'
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.23.1"
```

Comment:
This GH workflow is set to use Go `1.23.1` but the current versions used in packages are:
1. [1.22.5 for job-distributor protos](https://github.com/smartcontractkit/chainlink-protos/blob/f1b933b4d181ddeab31c17a3bd37f38abef59685/job-distributor/go.mod#L3C4-L3C10)
2. [1.22.5 for orchestrator protos](https://github.com/smartcontractkit/chainlink-protos/blob/f1b933b4d181ddeab31c17a3bd37f38abef59685/orchestrator/go.mod#L3)

I think that the easiest solution for now is to enforce a global go version an make sure that they're always in sync but in the future we might need a more elaborate solution

Reply:
ah yeap, this PR was created a week back, but we downgrade the golang version the other day, so it now different. I have downgraded it to the same version.

I dont think there is a builtin way to ensure all the packages use the go version, need some kind of script for it.